src/Data/ProgrammesDb/EntityRepository/Traits/ParentTreeWalkerTrait.php

```diff
@@ -114,12 +114,9 @@ protected function abstractResolveNestedAncestry(array $entities, string $key, c
     /**
      * Using the potential ancestors as a source, recursively set the parents into place
      *
-     * @param $entity
```

Comment:
well, I removed them because phpcs complains when you add a @param item without the type. Yeah, I will. :)